src/lib/Libutil/pbs_aes_encrypt.c

```diff
@@ -19,7 +19,7 @@ extern unsigned char pbs_aes_iv[];
 #define CIPHER_CONTEXT_CLEAN(v) EVP_CIPHER_CTX_cleanup(v)
 #else
 #define CIPHER_CONTEXT_INIT(v) v = EVP_CIPHER_CTX_new(); if (!v) return -1;
```

Comment:
@mike0042 Thanks for review... I have removed semicolon from definition of CIPHER_CONTEXT_INIT, please have a look in latest commit https://github.com/PBSPro/pbspro/pull/372/commits/4ac8775e90f23508fff10383dc245e97b079a090